API_Kraken/get_bitcon_price.py

```diff
@@ -0,0 +1,19 @@
+import requests
+import datetime
+
+get_request = requests.get('https://api.kraken.com/0/public/Ticker?pair=XBTUSD')
+price = get_request.json()['result']['XXBTZUSD']['a'][0]
+date = datetime.datetime.now().strftime("%c")
+value = ('Price of bitcon for: ' + date + ' is: ' + price + ' USD')
+try:
+    print(value)
+except:
+    print("An error has occured")
```

Comment:
i will be better to say what happend to the user and explain what inner exception is:

```
    except Exception as e
       raise Exception(f'"An error has occured due to: {e}')
```